admin_manual/configuration_files/big_file_upload_configuration.rst

```diff
@@ -65,7 +65,7 @@ for how to configure those values correctly:
 
 Apache
 ^^^^^^
-* `LimitRequestBody <https://httpd.apache.org/docs/current/en/mod/core.html#limitrequestbody>`_ (In Apache HTTP Server <=2.4.53 this defaulted to unlimited, but now defaults to 1 GiB. The new default limits uploads from non-chunking clients to 1 GiB. If this is a concern in your environment, override the new default by either manually setting it to ``0`` or to a value similar to that used for your local environment's PHP ``upload_max_filesize / post_max_size / memory_limit`` parameters.)
+* `LimitRequestBody <https://httpd.apache.org/docs/current/en/mod/core.html#limitrequestbody>`_ (In Apache HTTP Server <=2.4.53 this defaulted to unlimited, but now defaults to 1 GiB. The new default limits uploads from non-chunking clients to 1 GiB and also limits the maximum chunk size for recent versions of the official Nextcloud clients to 1 GiB even though they may require up to 5 GiB. This will break uploads >1 GiB under various circumstances. Our recommendation is to set this Apache parameter to 5 GiB or to the old default behavior of unlimited. Also make sure to review your local environment's PHP ``upload_max_filesize / post_max_size / memory_limit`` parameters to make sure they are at least the same size. For chunking capable clients that support manual configuration, as an alternative, the default maximum chunk size can be reduced to 1 GiB, but each client installation will need this parameter to then be manually configured.)
```

Comment:
It's 5GB today. It was changed in nextcloud/desktop#5939 

Context: The desktop client is more aggressive about using a large chunk size (versus our web client). This same behavior is essentially the underlying cause of nextcloud/desktop#4278 too (while many of those people are using CF since that's specifically covering the 100MB CF limit, the same underlying problem applies to body size limits in Apache and Nginx; only the absolute number varies).

Though today - rather than updating the docs - I'm of the opinion we should really fix the mentioned issue in the desktop client. Then the language in this PR can be more about "optimizing upload performance" rather than "fixing broken uploading".

Reply:
> Though today - rather than updating the docs - I'm of the opinion we should really fix the mentioned issue in the desktop client. Then the language in this PR can be more about "optimizing upload performance" rather than "fixing broken uploading".

Agreed 💯